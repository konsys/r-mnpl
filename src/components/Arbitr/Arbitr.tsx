import React from "react";
export const Arbitr = (props: any) => (
  <>
    <div className="table-body-board-center-arbitr">
      <input
        id="arbitr-pause"
        type="button"
        className="button button-small button-free button-grass"
        value="Приостановить игру"
      />
      <input
        id="arbitr-unpause"
        type="button"
        className="button button-small button-free button-grass"
        value="Возобновить игру"
        style={{ display: "none" }}
      />
      <input
        id="arbitr-uncontract"
        type="button"
        className="button button-small button-free button-grapefruit"
        value="Откатить договор"
        style={{ display: "none" }}
      />
    </div>
  </>
);
