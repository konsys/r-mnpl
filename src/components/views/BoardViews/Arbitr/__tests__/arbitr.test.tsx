import { mount, shallow } from "enzyme";

import { Arbitr } from "../Arbitr";
import React from "react";
import renderer from "react-test-renderer";

describe("A suite", function () {
  it("renders correctly", () => {
    const tree = renderer.create(<Arbitr />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders three inputs", () => {
    const wrapper = shallow(<Arbitr />);
    expect(wrapper.find("input")).toHaveLength(3);
  });
});
