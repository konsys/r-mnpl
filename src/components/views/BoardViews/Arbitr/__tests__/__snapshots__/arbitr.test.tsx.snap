// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite renders correctly 1`] = `
<div
  className="table-body-board-center-arbitr"
>
  <input
    className="button button-small button-free button-grass"
    id="arbitr-pause"
    type="button"
    value="Приостановить игру"
  />
  <input
    className="button button-small button-free button-grass"
    id="arbitr-unpause"
    style={
      Object {
        "display": "none",
      }
    }
    type="button"
    value="Возобновить игру"
  />
  <input
    className="button button-small button-free button-grapefruit"
    id="arbitr-uncontract"
    style={
      Object {
        "display": "none",
      }
    }
    type="button"
    value="Откатить договор"
  />
</div>
`;
